RELEASE-NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Wikibase DataModel Services release notes
 
+## Version 3.16.0 (dev)
```

Comment:
Just to confirm my understanding..
This is an interesting process .. so we add stuff under this that stays at `dev` until the actual release is made and then date would be put in here as a commit for the release as well right?

Reply:
This is what I have been doing since 2009 or so. What's interesting about this / what would you do instead? :)

Reply:
oh no I wouldn't do anything instead.. I find it interesting in a positive way.. I like it and would keep doing it.